src/tools/scripts/test_template_changes.sh

```diff
@@ -59,9 +59,9 @@ cat "${DIFF_FILENAME}"
 NUM_DIFFS=$(wc -l < "${DIFF_FILENAME}")
 if [ "${NUM_DIFFS}" -ne "0" ]; then
   # Weird syntax is to make is Mac compatible: https://stackoverflow.com/a/1252191/2144578
-  ESCAPED_OUTPUT=$(sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/%0A/g' -e 's/\r/%0D/g' -e 's/\%/%25/g' "${DIFF_FILENAME}")
+  ESCAPED_OUTPUT=$(sed -e ':a' -e 'N' -e '$!ba' -e 's/\%/%25/g' -e 's/\n/%0A/g' -e 's/\r/%0D/g' -e "s/'/%27/g" -e 's/"/%22/g' "${DIFF_FILENAME}")
```

Comment:
There are some replacements that can be done in Bash without sed. [See an example in the escapify function of a code I wrote a while ago](https://github.com/ibnesayeed/repo-attrs/blob/master/entrypoint.sh).